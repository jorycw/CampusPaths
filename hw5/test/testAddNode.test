# Testing file.
# Tests adding multiple nodes to a graph and listing them back as a 
# single line of nodes

# Create a graph
CreateGraph graph1

# Adds 5 different nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 startingWithDifLetter
AddNode graph1 aLetterBeforeN

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
