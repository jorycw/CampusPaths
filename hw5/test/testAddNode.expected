# Testing file.
# Tests adding multiple nodes to a graph and listing them back as a 
# single line of nodes

# Create a graph
created graph graph1

# Adds 5 different nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node startingWithDifLetter to graph1
added node aLetterBeforeN to graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: aLetterBeforeN n1 n2 n3 startingWithDifLetter